gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
The only way to not do this, is to have mocha be an independent task and not depend upon `scripts`. However, this then means we always need to do `gulp scripts && gulp mocha` when testing instead of just `gulp mocha` to run the unit tests. We'd also need to update the `test`  task to run scripts manually before mocha.